app/feature/feature-forever/src/main/kotlin/com/hedvig/android/feature/forever/ui/ForeverContent.kt

```diff
@@ -206,6 +217,9 @@ internal fun ForeverContent(
           coroutineScope.launch {
             editSheetState.hide()
             showEditBottomSheet = true
+            delay(400)
+            focusRequester.requestFocus()
+            textFieldValueState = textFieldValueState.copy(selection = TextRange(textFieldValueState.text.length))
```

Comment:
This highlights the entire thing right?

Reply:
Actually just sets the caret on the end of the textfield. I changed it so that it selects the text instead.